pkg/render/fluentd.go

```diff
@@ -126,23 +129,24 @@ type EksCloudwatchLogConfig struct {
 }
 
 type fluentdComponent struct {
-	lc              *operatorv1.LogCollector
-	esSecrets       []*corev1.Secret
-	esClusterConfig *ElasticsearchClusterConfig
-	s3Credential    *S3Credential
-	splkCredential  *SplunkCredential
-	filters         *FluentdFilters
-	eksConfig       *EksCloudwatchLogConfig
-	pullSecrets     []*corev1.Secret
-	installation    *operatorv1.InstallationSpec
-	clusterDomain   string
-	osType          OSType
-	image           string
+	lc                *operatorv1.LogCollector
+	managementCluster *operatorv1.ManagementCluster
```

Comment:
Is this value used for anything more than triggering static code? 
```suggestion
	isManagementCluster bool
```